_posts/2024-04-02-shared-conda-tutorial.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: The ideal setup for a multi-user conda installation
+tags: conda bash systems-administration
+contributors:
+- boulund
+- pmenzel
+- hexylena
+---
+
+[Conda](https://docs.conda.io/projects/conda/en/stable/) and [Bioconda](https://bioconda.github.io/) have become crucial tools in most bioinformatician's toolkit. It is often used to install applications and create reproducible environments on their system or a compute cluster. 
+
+On multi-user systems (e.g. shared Linux servers), multiple users can share  the same conda installation and the environments thererin. This is a good idea and usually works well when only already existing environments are re-used, but can become tricky when users also want to create environments (using `conda create`) into the shared conda installation. On a standard Linux system, this often results in permission problems when (re-)downloading packages and overwriting files that were created by others.
+
+Here we describe a setup with a shared (read-only) conda installation, with re-usable shared environments, and private per-user environments.
+
+
+## Tutorial
+
+### Creating the shared conda installation
+
+
+First, create a shared conda installation folder that is read-only for all regular users. Shared conda environments therein, which are available to all users, must be created by the sysadmin (or with a user account with the correct privileges for the conda folder). 
+
+In this tutorial we'll use `/apps/conda` as the path, but please choose something appropriate for your system.
+
+We install conda by downloading and running the [Miniconda installer](https://repo.anaconda.com/miniconda/), but feel free to use the [full Anaconda installer](https://repo.anaconda.com/archive/) if that is your preference. We also set up folder permissions and an initial conda config, which includes the recommended [bioconda](https://bioconda.github.io/) and [conda-forge](https://conda-forge.org/) channels.
+
```

Comment:
Ah, indirectly answered about distro here :)

Reply:
Do you want it to be more explicit?

Reply:
Yes please, just say the distros you know this work with straight up front.

Reply:
Well, I've personally only used it with Miniconda and Anaconda, so technically the two distros mentioned are the ones I know this works with. But I see your point. I also think it distracts from the scope of the post a bit to introduce a lot of choices and options at this stage. In my mind this post is really just a small and focused recipe for making a very run-of-the-mill standard shared conda installation. Users interested in other non-standard conda distributions probably know of some of the quirks of these already, and most likely won't need detailed hand-holding instructions anyway, don't you think?

Reply:
if we know those two to work, then I wouldn't mention any other and leave it like that.